docs/genai/tutorials/phi3-v.md

```diff
@@ -81,7 +79,6 @@ Support for Windows machines with GPUs other than NVIDIA is coming soon!
    * CUDA 12
 
    ```bash
-   pip install numpy
    pip install onnxruntime-genai-cuda --pre --index-url=https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/onnxruntime-cuda-12/pypi/simple/
```

Comment:
Can `--pre` be removed now that there are published stable releases that support Phi-3 vision?

Reply:
Fixed